commands/check_cloudformation.py

```diff
@@ -32,7 +32,12 @@ def check_cloudformation(ctx: click.Context) -> None:
 
 
 def get_lint_result(path: str):
-    command = ["cfn-lint", path]
+    command = [
+        "cfn-lint",
+        path,
+        "--ignore-templates",
+        f"{BASE_DIR}/tests/test-application/copilot/**/addons/addons.parameters.yml",
```

Comment:
Why are we ignoring this file?

Reply:
It is not a CloudFormation template file. `cfn-lint` will complain that it doesn't have `Resources` property and other errors/warnings.

Reply:
```suggestion
        # addons.parameters.yml is not a CloudFormation template file, so cfn-lint will complain that it doesn't have Resources property and other errors/warnings
        f"{BASE_DIR}/tests/test-application/copilot/**/addons/addons.parameters.yml",
```

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/107/commits/a46a1e8997c85d026985b5b3d42a1554f717eb90.